testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksUnityCompatibility.java

```diff
@@ -0,0 +1,218 @@
+                format("CREATE TABLE %s.%s.%s (c1 int, c2 string) using delta location '%s'",
+                        unityCatalogName, schemaName, sourceTableName, tableLocation2));
+        onDelta().executeQuery(
+                format("INSERT INTO %s.%s.%s values (1, 'one'), (2, 'two'), (3, 'three')",
+                        unityCatalogName, schemaName, sourceTableName));
+
+        onTrino().executeQuery(format("MERGE INTO delta.%s.%s t USING delta.%s.%s s on t.c1 = s.c1 " +
+                "WHEN MATCHED THEN UPDATE SET c2 = s.c2 " +
+                "WHEN NOT MATCHED THEN INSERT (c1, c2) VALUES (s.c1, s.c2)", schemaName, tableName, schemaName, sourceTableName));
+        assertThat(onTrino().executeQuery(format("SELECT * FROM delta.%s.%s", schemaName, tableName)))
+                .containsOnly(expectedRowsForMerge);
+        assertThat(onDelta().executeQuery(format("SELECT * FROM %s.%s.%s", unityCatalogName, schemaName, tableName)))
+                .containsOnly(expectedRowsForMerge);
+    }
+
+    @Test(groups = {DATABRICKS_UNITY_HTTP_HMS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testManagedSchemas()
+    {
+        String tableName = "test_managed_table";
+        onDelta().executeQuery(
+                format("CREATE TABLE %s.%s.%s (c1 int, c2 string)",
+                        unityCatalogName, schemaName, tableName));
+        assertThat(onTrino().executeQuery(format("SHOW CREATE SCHEMA delta.%s", schemaName)))
+                .containsOnly(row(format("CREATE SCHEMA delta.%s", schemaName)));
+        assertThat(onTrino().executeQuery("SHOW TABLES IN delta." + schemaName)).hasNoRows();
+    }
+
+    @Test(groups = {DATABRICKS_UNITY_HTTP_HMS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testTableColumns()
+    {
+        String tableName = "test_cols";
+        String tableLocation = format("%s/%s/%s", externalLocationPath, schemaName, tableName);
+        onDelta().executeQuery(format("CREATE TABLE %s.%s.%s (" +
+                "int_col INT," +
+                "string_col STRING," +
+                "tinyint_col TINYINT," +
+                "smallint_col SMALLINT," +
+                "bigint_col BIGINT," +
+                "decimal_col DECIMAL," +
+                "float_col FLOAT," +
+                "double_col DOUBLE," +
+                "date_col DATE," +
+                "timestamp_col TIMESTAMP," +
+                "binary_col BINARY," +
+                "bool_col BOOLEAN," +
+                "array_int_col ARRAY<int>," +
+                "map_col MAP<TIMESTAMP,INT>," +
+                "struct_col struct<a: LONG, b: String NOT NULL>" +
+                ") " +
+                "USING DELTA " +
+                "LOCATION '%s'", unityCatalogName, schemaName, tableName, tableLocation));
```

Comment:
Indentation seems wrong.

Reply:
I tried to indent it to left and the check-style plugin started complaining.